doc/Translation System.md

```diff
@@ -0,0 +1,54 @@
+<%=translation('yaml key to string',{},false)%>
+```
+You can find calls like this in views that have translation function call in search bars and buttons.
+
+Some improvements to this call can be reorganising the parameters to make it bit concise but it is tricky as lots of tests are written that follow the existing call structure, many function calls have this structure and also the gem has an inbuilt options parameter that can be passsed as array of options or as list of key value pairs , so for now, this call is simple to check and for now though a bit longer.
+
+## Importing and exporting translations 
+Moving on to the second part of the project. We use [Transifex](https://www.transifex.com/publiclab/publiclaborg/dashboard/) localisation platform to handle voluntary contributions and add them regualarly to the code-base. 
+### Importing translation
+We don't need to exclusively add new strings to the Transifex project.Adding it as a key-value pair in the `en.yml` file will do the trick. Transifex keeps track of the file and automatically adds new strigs at regular intervals. Managers and admins can also run a manual sync from `settings->integrations->Send to Github`.  option, more about it [here](https://publiclab.org/notes/ajitmujumdar25999/07-18-2021/importing-new-translations-from-transifex-project). Once imported you can find the string in the Transifex resource file. You can check by searching it in the translation search bar.
+### Exporting translations 
+ Transifex bot automatically raises a PR when a language resource is 100% reviewed. Managers and admins can also trigger a manual sync by `settings->integrations->Send to Github` option, we can specify a threshold percentage and any language files having reviewed percentage above the threshold will be added and a PR will be raised by the Transifex bot. Alternatively, we can also directly download a yaml file for the translation resource from the transifex site. More about it [here](https://publiclab.org/notes/ajitmujumdar25999/07-18-2021/importing-new-translations-from-transifex-project)
+
+## Translation life-cycle
+So now that you know all the aspects in this project, let's see how the entire project works. 
+### Step 1: Adding a string
+Add new strings as a key value pair in the `en.yml` file.
+### Step 2: Importing string to Transifex project 
+Transifex automatically fetches new strings from `en.yml` file. Managers and admins can also run manual sync to add strings to Project.
+### Step 3: Translating string
+Translation team members can add translations on the Transifex site. You can find a detailed wiki on how to do so [here](https://publiclab.org/wiki/translation?_=1629559376#Activities+for+people+who+want+to+translate+this+website)
+### Step 4: Exporting translated strings
+Once the string is reviewed,it can be added back to the Github repo. Transifex bot automatically raises a PR once language resource is 100% reviewed. Managers and admins can also trigger a manual sync.
```

Comment:
Can we add links and potentially screenshots of where to trigger a manual sync, as well as a link to an example bot PR?

Reply:
okay I'll add some screenshots